docs/server/source/appendices/log-rotation.md

```diff
@@ -0,0 +1,46 @@
+Copyright BigchainDB GmbH and BigchainDB contributors
```

Comment:
The block at the top should be an HTML comment (so it doesn't render on ReadTheDocs). To do that, you will have to add `<!--` or `<!---` before it, maybe as a new line 1. (All HTML markup is valid Markdown markup.)